folly/Traits.h

```diff
@@ -384,10 +384,24 @@ struct Bools {
   }
 };
 
+template <bool... Bs>
+struct AllFalse : std::is_same<Bools<Bs...>, Bools<(Bs && false)...>> {};
```

Comment:
Asking @ericniebler because this could affect other code in folly which he knows about which actually depends on the current deferred evaluation semantics of `StrictConjunction`.